calculate_movement_method.py

```diff
@@ -3,11 +3,23 @@
 import boto3
 import pandas as pd
 from es_aws_functions import general_functions
+from marshmallow import Schema, fields
 
 import imputation_functions as imp_func
 
-lambda_client = boto3.client('lambda', region_name='eu-west-2')
-s3 = boto3.resource('s3')
+
+class RuntimeSchema(Schema):
+    movement_type = fields.Str(required=True)
+    data = fields.List(fields.Dict, required=True)
+    questions_list = fields.List(fields.String, required=True)
+    current_period = fields.Str(required=True)
+    period_column = fields.Str(required=True)
+    previous_period = fields.Str(required=True)
+
+
+# Why is this here. It isn't in any other section????????????????
```

Comment:
Ironically, why is this comment here?

Reply:
In this file and in no other file the following code is called outside the main function but in all others it is called inside of it. I wrote that to remind me to ask if there was a special reason or if for standards sake we could move it to match.

```
lambda_client = boto3.client("lambda", region_name="eu-west-2")
s3 = boto3.resource("s3")
```

Reply:
This is the method. Boto3 things aren't needed in the method and are all done in the Wrangler as such I believe this can be completely removed.